compiler/src/printCommon.ml

```diff
@@ -17,6 +17,11 @@ let pp_aligned fmt =
   function
   | Memory_model.Aligned ->
      Format.fprintf fmt "#aligned "
+  | Unaligned -> ()
```

Comment:
I think the function pp_aligned is not used anymore.
So it can be removed.

Reply:
I think you are wrong.